Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```diff
@@ -204,6 +204,70 @@
     interactFailureString: petting-failure-honkbot
     interactSuccessSound:
       path: /Audio/Items/bikehorn.ogg
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 4
+    baseSprintSpeed: 6
+
+- type: entity
+  parent: MobSiliconBase
+  id: MobSyndieBot
+  name: syndiebot
+  description: Death to Nanotrasen!
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Silicon/Bots/syndiebot.rsi
+    state: syndiebot
+  - type: GhostRole
+    makeSentient: true
+    name: ghost-role-information-syndiebot-name
+    description: ghost-role-information-syndiebot-description
+  - type: GhostTakeoverAvailable
+  - type: InnateTool
+    tools:
+      - id: WeaponPulsePistol
+      - id: EnergyDagger
```

Comment:
12 seems cheaper for something with an edagger and takes 12 punches to kill.

Reply:
edagger 2 tc or something
cobra 3 tc rough equivalent
5 tc of weaponry + cant be stunned + very fragile + needs a ghost = 12 tc ??????

Reply:
> 12 seems cheaper for something with an edagger and takes 12 punches to kill.

E-Dagger costs 2 TC, it can be broken in 6 punches with a crowbar.

Reply:
I think we should lower the price, for example, to 10 TC. He can be stunned, he is fragile, he has weapons for, say, 5 TC.